icepyx/tests/test_behind_NSIDC_API_login.py

```diff
@@ -12,20 +12,25 @@
 
 @pytest.fixture
 def reg(scope="module"):
```

Comment:
Scope should be added to the pytest decorator.

```suggestion
@pytest.fixture(scope="module")
def reg():
```